package.json

```
@@ -49,7 +49,7 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "dependencies": {
-    "@babel/parser": "7.16.4",
+    "@babel/parser": "7.21.3",
```

```suggestion
    "@babel/parser": "7.22.5",
```

Maybe update to the latest.

---

Thank you. I did this manually to update lock file as well.